### PR TITLE
added "public Throwable getError()" and rewrote "void result()"

### PR DESCRIPTION
**Pull request by imixac**
added "public Throwable getError()" and rewrote "void result()"

### CODE REVIEW

File: src/main/java/ru/yandex/qatools/allure/cucumberjvm/AllureReporter.java [hunk 1]
```diff
@@ -242,6 +242,10 @@ public void result(Result result) {
         }
     }
 
+    public Throwable getError(Result result){
```
Comment (clicman) [inline src/main/java/ru/yandex/qatools/allure/cucumberjvm/AllureReporter.java]: Add Javadoc description. Point what it used for error overrides

**Review by clicman — CHANGES_REQUESTED** [review]

File: src/main/java/ru/yandex/qatools/allure/cucumberjvm/AllureReporter.java [hunk 2]
```diff
@@ -242,6 +242,17 @@ public void result(Result result) {
...
     }
 
+    /**
+     * Returns Throwable exception
+     * It can be overriden in extended Reporter
+     *
+     * @param result
+     * @return Returns Throwable exception
+     */
+    public Throwable getError(Result result){
```
Comment (baev) [inline src/main/java/ru/yandex/qatools/allure/cucumberjvm/AllureReporter.java]: Seems like you can use `protected` instead

**Review by baev — CHANGES_REQUESTED** [review]

**Review by baev — APPROVED** [review]

**Comment by imixac** [comment]
Merge?

**Comment by baev** [comment]
yeah, as soon as @clicman approve

**Review by clicman — APPROVED** [review]

**Comment by baev** [comment]
@imixac thanks!

**Comment by baev** [comment]
@imixac should I release the change or you have more incoming patches?

**Comment by clicman** [comment]
@baev There is no more planned changes. The @imixac is my teammate. So it could be released right now.

**Comment by baev** [comment]
@clicman could you please add release notes?

**Comment by clicman** [comment]
@baev Done https://github.com/allure-framework/allure-cucumberjvm/releases/tag/1.6.4

**Pull request merged**